packages/serve/src/lib/middleware/response-format/helpers.ts

```diff
@@ -29,6 +29,8 @@ export const checkUsableFormat = ({
 
   // if path ending has no format
   if (!pathFormat) {
+    // Use the default when "acceptFormat" us array type, because array type only happened when "supportedFormats" is [] and "Accept" header has multiple
+    if (Array.isArray(acceptFormat)) return defaultFormat;
```

Comment:
```suggestion
    if (supportedFormats.length === 0) return defaultFormat;
```
Can we check `supportedFormats` instead? When we didn't set any supportedFormat, return the default value. It's more understandable.

Reply:
Hi @oscar60310, thanks for suggestion, it has been fixed, please check it, thanks